api/specs/web-server/_projects_crud.py

```diff
@@ -59,7 +58,16 @@ async def create_project(
     "/projects",
     response_model=Page[ProjectListItem],
 )
-async def list_projects(_params: Annotated[ProjectListParams, Depends()]):
+async def list_projects(
+    _params: Annotated[ProjectListParams, Depends()],
+    order_by: Annotated[
+        Json | None,
+        Query(
+            description="Order by field (type|uuid|name|description|prj_owner|creation_date|last_change_date) and direction (asc|desc). The default sorting order is ascending.",
+            example='{"field": "prj_owner", "direction": "desc"}',
+        ),
+    ] = None,
```

Comment:
since the code orders by default by "last_change_date" this should be here as well the default. then you can remove that if-else case you have in _crud_handlers